Cargo.toml

```diff
@@ -21,8 +21,16 @@ path = "regex_macros/benches/bench_dynamic.rs"
 test = false
 bench = true
 
+[dependencies.regex-syntax]
+path = "regex_syntax"
+version = "*"
```

Comment:
I personally prefer to write out this section like so nowadays:

```toml
[dependencies]
regex-syntax = { path = "regex_syntax", version = "*" }
```

Also, perhaps this could have a more strict version requirement than `*`? And finally, perhaps the folder could be called `regex-syntax` to match the crate name?